src/config.js

```diff
@@ -203,6 +203,41 @@ export const BENCHMARKS = {
     labels: ['Firefox (tiering)', 'Firefox (wasm-baseline)', 'Firefox (wasm-ion)', 'Chrome'],
     label: 'WebAssembly Embenchen',
   },
+  'wasm-godot': {
+    compare: {
+      'raptor-wasm-godot-firefox': {
+        color: '#e55525',
+        label: 'Firefox (tiering)',
+        frameworkId: RAPTOR_FRAMEWORK_ID,
+        suite: 'raptor-wasm-godot-firefox',
+        buildType: 'opt',
+      },
+      'raptor-wasm-godot-baseline-firefox': {
+        color: 'red',
+        label: 'Firefox (wasm-baseline)',
+        frameworkId: RAPTOR_FRAMEWORK_ID,
+        suite: 'raptor-wasm-godot-baseline-firefox',
```

Comment:
At the moment I don't see any scores for `baseline` and for `ion`.
Once they show up on Treeherder/Perfherder they will start showing up.

Reply:
Sounds good to me too!